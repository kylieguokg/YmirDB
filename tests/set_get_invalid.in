get dang
set dang dang
get dang
set out in
get out
set a 
get a
set
list entries
bye 
