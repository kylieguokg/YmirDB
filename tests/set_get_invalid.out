> no such key

> not permitted

> no such key

> no such key

> no such key

> ok

> []

> invalid key

> a []

> bye
